### DISCUSSION TITLE
Is it possible to add additional .dylib files to ipa export

### DISCUSSION THREAD

**Discussion by fatduckling** [godotengine/godot-proposals#8529] [Scripting]
Hi,

In my `.gdextension` file, I have my main game library defined like:
```
ios.arm64 = "res://libGameLibrary_ios.dylib"
```
When I export the game to iOS IPA file, this dylib file gets packaged in the IPA file, which is great.

However, when my `libGameLibrary_ios.dylib` has dependencies on other libraries, such as `libtbb.12.dylib`, `libssl.2.dylib`, `libcrypto.2.dylib`, etc, is there any way to include these additional libraries in the IPA file as well?

I could manually open the generated XCode project and add the additional libraries like I'm doing now, but ideally I'd like to automate this process so I don't have to manually add the project on every build.

For other platforms it's not an issue because I can just copy the required shared libraries to the executable folder - it's only an issue for IPA files.

Any suggestions?

**Comment by fatduckling** [comment]
Apparently this is possible with the `[dependencies]` section. Will look into it now, thanks

**Comment by bruvzg** [comment]
You can use `[dependencies]` section of the `.gdextension` file.

```ini
[dependencies]
ios.arm64 = {
  "res://bin/libtbb.12.dylib": "",
  "res://bin/libssl.2.dylib": ""
}
```

Note: second part of each pair is target path, but it's used only on macOS and should be empty on other platforms.

**Reply by fatduckling** [reply]
Thanks very much, this works perfectly! Much appreciated ❤️
